rules/REQUEST-921-PROTOCOL-ATTACK.conf

```diff
@@ -30,7 +30,7 @@ SecRule TX:EXECUTING_PARANOIA_LEVEL "@lt 1" "id:921012,phase:2,pass,nolog,skipAf
 # [ References ]
 # http://projects.webappsec.org/HTTP-Request-Smuggling
 #
-SecRule ARGS_NAMES|ARGS|REQUEST_BODY|XML:/* "@rx (?:get|post|head|options|connect|put|delete|trace|track|patch|propfind|propatch|mkcol|copy|move|lock|unlock)\s+(?:/|\w)[^\s]*(?:\s+http/\d|[\r\n])" \
+SecRule ARGS_NAMES|ARGS|REQUEST_BODY|XML:/* "@rx (?:get|post|head|options|connect|put|delete|trace|track|patch|propfind|propatch|mkcol|copy|move|lock|unlock)\s+(?:([\r\n]|\/\w|\/|http:\/\/))" \
```

Comment:
I believe the presence of '\/' in that pattern makes '\/\w' redundant since the latter is a subset.
The extra () add a capture group and are not needed.

Reply:
Agreed to both points.

Reply:
Thanks - I've tried your suggestion - the [5th test case](https://github.com/coreruleset/coreruleset/blob/24cc89e37b62ca3884f39b234195fa95c5540560/tests/regression/tests/REQUEST-921-PROTOCOL-ATTACK/921110.yaml#L76-L92) is failed (which is a negative test).

See this [regex101](https://regex101.com/r/1s7754/1) case - the
```
var=aaa
get /foo
```
shouldn't matched.